docs/install/azure.md

```diff
@@ -0,0 +1,227 @@
+(install-azure)=
+
+# Installing on Azure
+
+## Goal
+
+By the end of this tutorial, you should have a JupyterHub with some admin
+users and a user environment with packages you want to be installed running on
+[Microsoft Azure](https://azure.microsoft.com).
+
+This tutorial leads you step-by-step for you to manually deploy your own JupyterHub on Azure cloud.
+
+:::{note}
+✨ The `Deploy to Azure button` project allows you to deploy your own JupyterHub with minimal manual configuration steps. The deploy to Azure button allows you to have a vanilla configuration in just one-click and by assigning some variables.
+
+Check it out at [https://github.com/trallard/TLJH-azure-button](https://github.com/trallard/TLJH-azure-button).
+:::
+
+## Prerequisites
+
+- A Microsoft Azure account.
+- To get started you can get a free account which includes 150 dollars worth of Azure credits ([get a free account here](https://azure.microsoft.com/en-us/free//?wt.mc_id=TLJH-github-taallard))
+
+These instructions cover how to set up a Virtual Machine
+on Microsoft Azure. For subsequent information about creating
+your JupyterHub and configuring it, see [The Littlest JupyterHub guide](https://the-littlest-jupyterhub.readthedocs.io/en/latest/).
+
+## Step 1: Installing The Littlest JupyterHub
+
+We start by creating the Virtual Machine in which we can run TLJH (The Littlest JupyterHub).
+
+1.  Go to [Azure portal](https://portal.azure.com/) and login with your Azure account.
+
+2.  Expand the left-hand panel by clicking on the ">>" button on the top left corner of your dashboard. Find the Virtual Machines tab and click on it.
+
+    ```{image} ../images/providers/azure/azure-vms.png
+    :alt: Virtual machines on Azure portal
+    ```
+
+3.  Click **+ add** to create a new Virtual Machine
+
+    ```{image} ../images/providers/azure/add-vm.png
+    :alt: Add a new virtual machine
+    ```
+
+4.  Select **Create VM from Marketplace** in the next screen.
+    A new screen with all the options for Virtual Machines in Azure will displayed.
+
+        ```{image} ../images/providers/azure/create-vm.png
+        :alt: Create VM from the marketplace
+        ```
```

Comment:
````suggestion
    ```{image} ../images/providers/azure/create-vm.png
    :alt: Create VM from the marketplace
    ```
````